exercises/exercise_4/README.md

```diff
@@ -1,248 +1,190 @@
-
-You can of course use the same flags `--set` and `--parameters-files` as with the `render` command.
-
-* **`install`** another `hello` application, modifying the ports and the installation name
-
-```sh
-$ docker-app install hello.dockerapp --name my-hello-prod-installation -f prod-parameters.yml
-Creating network my-hello-prod-installation_default
-Creating service my-hello-prod-installation_hello
+# Exercise 4 - Configure your Application with Parameters
 
-$ docker stack ls
-NAME                         SERVICES            ORCHESTRATOR
-my-hello-installation        1                   Swarm
-my-hello-prod-installation   1                   Swarm
-```
+> **Time**: Approximately 15 minutes
+>
+> **Difficulty**: Easy
 
-**NOTE:** `install` can take any dockerapp single-file or directory. In that case it will automatically bundle the application then install it. `install` can also deploy directly a previously bundled `bundle.json`.
+## Table of Contents
 
-* **`install`** using the bundle.json previously built
+1. [Docker App Parameters](#docker-app-parameters)
+1. [Using a Parameter to Change Voting Options](#using-a-parameter-to-change-voting-options)
+1. [Upgrading a Deployed Application](#upgrading-a-deployed-application)
 
-```sh
-$ docker-app install bundle.json --name my-app-with-bundle -s hello.port=8181
-Creating network my-app-with-bundle_default
-Creating service my-app-with-bundle_hello
-```
+## Exercise Objectives
 
-### Status
+By the end of this exercise, you will have:
 
-`Usage:  docker-app status <installation-name> [flags]`
+- Learned about Docker App parameters
+- Added parameters to the voting app application
+- Learned how to deploy an update to an already-deployed application
 
-`status` command takes an installation name and displays informations about deployment or upgrade, service per service.
 
-* **Display** the status of each installation
 
-```sh
-$ docker-app status my-hello-installation
-ID                  NAME                          MODE                REPLICAS            IMAGE                        PORTS
-qspyvbdxf1js        my-hello-installation_hello   replicated          1/1                 hashicorp/http-echo:latest   *:8080->8080/tcp
+## Docker App Parameters
 
-$ docker-app status my-hello-prod-installation
-ID                  NAME                               MODE                REPLICAS            IMAGE                        PORTS
-5d1glvey8b1l        my-hello-prod-installation_hello   replicated          1/1                 hashicorp/http-echo:latest   *:80->80/tcp
-```
+In addition to using environment variables, Docker App allows you to define parameters for an application that can be overridden at deploy-time. Examples of parameters might include port mappings, label definitions, or network names. _Almost_ everything in the Docker App compose file can be parameterized, except for the image names.
```

Comment:
Add also `replicas or CPU limit` in the example list, as they are most common use cases?
And maybe put the sentence `except for the image names` in bold 😄